docs/running-locally.md

```diff
@@ -154,6 +154,21 @@ To test SSO integration, use `PROFILE=sso`:
 make start UI=true PROFILE=sso
 ```
 
+## TLS
+
+By default, `make start` will start Argo in [plain text mode](./tls.md#plain-text).
```

Comment:
```suggestion
By default, `make start` will start Argo in [plain text mode](tls.md#plain-text).
```

`./` isn't needed and isn't used in docs links

Reply:
Okay, done in c430b3c7e0a3a6159270d939f945c06ea1c7594a